upgrading/automated_upgrades.adoc

```diff
@@ -732,6 +732,23 @@ other nodes in their class (`app` versus `infra`), one at a time.
 
 . Upgrade the {product-title} nodes running GlusterFS one at a time.
 
+.. Add the following parameters in the inventory file at *_/etc/ansible/hosts_*:
++
+----
+openshift_storage_glusterfs_version=v<your_cns_version> <1>
+openshift_storage_glusterfs_block_version=v<your_cns_version>
+openshift_storage_glusterfs_heketi_version=v<your_cns_version>
+----
+<1> Specify the CNS version, such as 3.9 or 3.11.3.
++
+.. Update the image tags from latest to <your_cns_version> (such as 3.9 or 3.11.3) on the following resources:
```

Comment:
I defer to QE on this, but it's not clear from a cold read of the step and the codeblock what you're supposed to do.
```suggestion
.. Update the image tags from *latest* to `<your_cns_version>`, such as *3.9* or *3.11.3*, on the following resources:
```

Reply:
I agree with your change. The term "latest" in this context is an actual choice, not a general term.